python-threatexchange/threatexchange/descriptor.py

```diff
@@ -68,8 +68,7 @@ def is_false_positive(self) -> bool:
     @property
     def is_mine(self) -> bool:
         """This Descriptor is my App's Opinion"""
-        # TODO - come up with a way to do this that doesn't use class state
```

Comment:
comment is still relevant or no?

Reply:
Still using class state :/

Reply:
Moved the comment, but still functionally similar